spotbugs/src/main/java/edu/umd/cs/findbugs/detect/FindDeadLocalStores.java

```diff
@@ -209,6 +209,10 @@ private void analyzeMethod(ClassContext classContext, Method method) throws Data
             return;
         }
 
+        if (method.getName().startsWith("$$_hibernate_")) {
```

Comment:
Might be worth ignoring other `$$` methods?

Reply:
I thought about it by I think the underlying issue might very well be a bug in Hibernate's bytecode enhancer, to at least it seems to be producing suboptimal bytecode here.
In case other false positives are reported it will be easy to ignore more methods